README.md

```diff
@@ -104,6 +104,10 @@ Run the schema generator via `node main.js`.
 
 -o, --out
     Specify the output file path. Without this option, the generator logs the response in the console.
+
+--allow-additional-properties  <true|false>
```

Comment:
Can we find a different name here? `allow-additional-properties: false` to me sounds like you cannot use additional properties at all. Maybe `additional-properties-by-default` or just `additional-properties` (with a precise description) are better. What do you think?

Reply:
`additional-properties` sounds good to me

Reply:
Let's also rename the variable used in the code.

Reply:
All done, test and build are both passing